img/s/skinport.com.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 731 88.1"><path fill="#fa490a" d="M150.8 1.9h21.9v21.9h-21.9z"/><path fill="#fff" d="M202.8 1.9h18.9v84.5h-18.9V1.9zm115.7 54.4L277 1.9h-17.6v84.5h18.9V33.5l40.3 52.9h18.9V1.9h-18.9v54.4zM41.8 35.7c-8-2-13.1-3.7-15.4-5.1-2.2-1.5-3.3-3.4-3.3-5.9 0-2.5.9-4.4 2.7-5.9 1.8-1.5 4.3-2.2 7.4-2.2 7.8 0 15.4 2.8 22.7 8.3l9.5-13.8c-4.2-3.5-9.1-6.3-14.7-8.2C45.1 1 39.5 0 33.8 0 25.2 0 18 2.2 12.3 6.5 6.5 10.9 3.7 17 3.7 25.1c0 8 2.3 13.9 6.8 17.6 4.6 3.7 11.7 6.8 21.6 9.1 6.2 1.5 10.4 3 12.5 4.5 2.1 1.5 3.1 3.5 3.1 6s-1 4.6-3 6c-2 1.5-4.7 2.2-8.2 2.2-7.7 0-16.2-4-25.3-12L0 72.5c10.7 9.9 22.8 14.9 36.1 14.9 9.3 0 16.7-2.4 22.4-7.1 5.7-4.7 8.5-10.9 8.5-18.4 0-7.6-2.2-13.3-6.6-17.3-4.4-3.9-10.6-6.9-18.6-8.9zM547 44.3c0-24.4-19.5-44-43.9-44-24.3 0-43.9 19.6-43.8 44 0 24.3 19.6 43.8 43.8 43.7 24.3.1 43.9-19.5 43.9-43.7zm-18.4-.2c.1 14.1-11.5 25.8-25.5 25.8s-25.5-11.7-25.4-25.8c.1-14 11.5-25.4 25.5-25.4 14-.1 25.4 11.3 25.4 25.4zM731 2h-68.5v16.6h25.8v67.8h18.9V18.6H731V2zM441.2 32.3c0 20.4-13.2 32.1-35 32.1h-12.3v22H375V2h30.1c22.5 0 36.1 11 36.1 30.3zm-18.1.9c0-6.4-3-14.6-19.4-14.6H394v29.3h10.8c14.6 0 18.3-8.3 18.3-14.7zm-303.9 8.2l32.2 45h22.8l-32.2-45h-22.8zM100.3 1.9v84.5h18.9V1.9h-18.9zm521.9 84.5l-15.8-22h-13.3v22h-18.9V1.9l30.2.1c22.4 0 36 11 36 30.3 0 13.1-5.4 22.6-15.1 27.7l18.9 26.4h-22zm-29-38.5H604c14.7 0 18.3-8.3 18.3-14.6s-3-14.6-19.4-14.6h-9.7v29.2z"/></svg>
```

Comment:
This might be a better logo:
![skinport com](https://user-images.githubusercontent.com/520841/171160456-bb5534df-c6f2-4b2d-a6ab-bcda5bbbcce0.svg)
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" xml:space="preserve" viewBox="0 0 263.1 217.7"><path fill="#fa490a" d="M113.5 202.8s-14.7 10.2-35.9 7.8c0 0 3.1-4.6-3.8-15.9-7.5-12.2-19.9-28.5-21.3-43.1z" class="st0"/><path fill="#016699" d="M241.9 113.8c11.3 47.8 0 47.2-6.7 69.4-6.7 22.2-51.6 41.5-105.9 32-48.6-8.5-85.5-64.5-86-78.4 96.2 43.4 198.1-22.7 198.6-23z"/><path fill="#fa490a" d="M184.2 48.7S158 8.5 112.5 0c0 0 14.6 15.5 8.4 21.7-3.2 3.2-12.3-3.2-17.1-9.4 0 0 3.3 10.3-15.8 28.5-10.2 9.8-37.1 35.8-37.2 67.8Zm-141 88.1S40.4 99.2 0 88.3c0 0 .1 29.2 23.4 46.3 0 0-16.7 16.7-8 40.6 0 .1 26.4-4.5 27.8-38.4z" class="st0"/><path fill="#4db5da" d="M43.2 136.8c-.5-13.9 8-65.8 64.7-92.2 56.7-26.4 117.6 11.5 132.4 63.1l1.6 6.1c-.5.3-107.8 76.4-198.7 23z" class="st2"/><path fill="#01324d" d="M147.1 214.6c16.4-4.7 28.3-39.3 29.5-42.4-4.9 3.6-10-.9-6.6-6l-2.8.1c-2.1 0-4 .2-5.9.5-4.9.8-8.6 2.5-13 4.5l-6.7 2.9a26 26 0 0 1-5.3 1.6c-3.3.6-4.9-.1-5.3-.3l-4.9-4.5s-3.3 5.9-4.3 8a28.9 28.9 0 0 0 3.1 29.1c4.1 5 13.5 9 22.2 6.5z" class="st3"/><path fill="#225d73" d="M225.9 122.8c-10.6-35.2-28-72.9-44-79.6a44 44 0 0 0-6.9-.6l-3 .1a53 53 0 0 0-36.7 22.4c-14.7 19.1-24.1 47.8-21.9 61.2 1.6 10 7.3 18.3 15.3 24.7 1.7 1.3 3.4 2.6 5.2 3.7 14.2 1.3 31.1-2.6 43.8-6.7 21.2-6.7 36.5-17.9 48.3-24.6l-.1-.6z"/><path fill="#01324d" d="m230.3 192.1.7-1 .7-1 .6-.9.7-1.1.5-.8.6-1.3.3-.7.7-2c2.8-9.4-1.3-33.8-9-59.7-19 10.7-53.2 27-92.2 31.2l1.7 1c13.5 7.9 43.8 3.2 47.6 9.9 3.9 6.6-8.4 11.3-2.4 20.3a16 16 0 0 0 9.3 6.6c.1 4.1 1.1 8.3 3.6 11.7 2.3 3.1 5.5 5.1 9.2 5.8l2.8-1.1.4-.2 2.4-1.1.7-.3 2.2-1.1.8-.4 2-1.1.8-.5 1.8-1 .9-.5 1.6-1 .9-.6 1.5-1 .9-.7 1.3-1 .9-.7 1.2-1 .9-.8 1-1 .8-.9.9-1 .7-1z" class="st3"/><path fill="#fa490a" d="M176.6 172.2s-11.9 31.6-29 35.8c-17.1 4.2-27.6-23.7-21.5-39.4 0 0 4.9 4.7 16.8 1.8 11.9-2.9 16.1-5.8 27.1-4.2 0 0 .8 5.8 6.6 6z" class="st0"/><path fill="#fff" d="M173.1 42.1c15.2-1.4 51.9 12.3 73 51.5s21 86.9 8.3 103.6c-12.7 16.7-10.4-10.1-10.4-10.1s-5.1 17-15.5 18.3c-10.4 1.3-5.9-17.5-5.9-17.5s-5.2 13.5-13.3 13.9c-8.1.4-8.5-10.4-5.5-20.2 0 0-9.4 4.5-13.4-1.5s4.9-10.2 5.1-15.9c.2-5.7-5.7-13.1-25.8-12.8-20 .3-44.4-15-47.5-33.9-3.1-18.9 15.7-72.1 50.9-75.4z"/><path fill="#232728" d="M219.6 141.5c11.4-4.7 19.1-24.9 8-44.7-11.1-19.8-33.4-17.8-46.9-8.3 0 0 8.5-1.8 17.2 2.5 0 0-13.2 4.9-12.8 22 .4 17.1 17 35.6 34.5 28.5zm28.6-7c3.8 1.1 6.4 18.6-1.1 18.2-7.5-.3-6.7-20.6 1.1-18.2z" class="st6"/><path fill="#4db5da" d="M221 105.3c5.5 2.2 5.2 16.6-3.7 16.6-12 0-6.8-20.8 3.7-16.6z" class="st2"/></svg>
```